action.yml

```
@@ -0,0 +1,34 @@
+name: "Infracost Actions"
+description: See cloud cost estimates for Terraform in pull requests. Requires hashicorp/setup-terraform@v1.
```

```suggestion
description: See cloud cost estimates for Terraform in pull requests.
```

I think this goes in the marketplace description so we don't want that

---

I moved it to:
```
inputs:
  path:
    description: The path that will be passed to `infracost breakdown`. This may be a path to a Terraform plan JSON or a Terraform project. Project support requires Terraform to have been installed with hashicorp/setup-terraform@v1.
    required: true
```
Is that ok or should we scratch it completely?